sql/src/SqlCsv.sk

```diff
@@ -355,6 +417,12 @@ fun applyDiffStrategy(
       tableName = table.name.origName;
       authorCol = SKDB.getAuthorIdx(table);
       accessCol = SKDB.getAccessIdx(table);
+      rows = sourceSchema match {
```

Comment:
Should we just pass this in to applyDiffStrategy? Stops the interface growing and helps keep the complexity of this down.

Reply:
I think not -- we need a (mutable) context to do this translation (to compute default values), so pushing it down into applyDiffStrategy makes sure that context transformations are produced/applied as expected.

Alternatively, we could use the mutable context from `replayDiff` to translate these rows, but I was worried that would interfere with the context juggling happening here.